dev/build.image/profiles/jakartaee9/features.xml

```diff
@@ -1,5 +1,11 @@
-<feature>jakartaee-9.0</feature>
-<feature>jakartaeeClient-9.0</feature>
+<feature>jakartaee-9.1</feature>
+<feature>jakartaeeClient-9.1</feature>
 <feature>appSecurityClient-1.0</feature>
+<feature>monitor-1.0</feature>
+<feature>ldapRegistry-3.0</feature>
+<feature>appSecurity-4.0</feature>
```

Comment:
appSecurity-4.0 is not required since it is already part of the convenience feature.